statscache/app.py

```diff
@@ -36,24 +28,36 @@ def main(name):
     )
 
 
+@app.route('/')
+def index():
+    """ Generate a JSON-P response with an index of plugins (as an array) """
+    return jsonp(json.dumps(plugins.keys()), 200)
```

Comment:
How about switching the response content-type based on the HTTP Accept header? That will allow us to use the same endpoint for JSON, HTML, CSV, and whatever other serialization formats that we decide to include later.